pandas/io/formats/style_render.py

```diff
@@ -477,6 +494,108 @@ def _generate_col_header_row(self, iter: tuple, max_cols: int, col_lengths: dict
 
         return index_blanks + column_name + column_headers
 
+    def _generate_descriptor_row(self, iter: tuple, max_cols: int):
```

Comment:
What kind of tuples are allowed? Would be better to be as specific as possible to reduce any future refactor risks.